package.json

```diff
@@ -57,7 +57,7 @@
     "argon2": "^0.21.0",
     "concolor": "^0.1.11",
     "csv-stringify": "^5.3.0",
-    "globalstorage": "^0.7.0",
+    "globalstorage": "^0.7.2",
```

Comment:
You should probably update to `^0.7.3`.

Reply:
Done by @belochub here https://github.com/metarhia/impress/pull/1145